### ISSUE TITLE
eeglab downloads 357.7MB for xdfimport plugin

### ISSUE THREAD

**Issue by cboulay** [sccn/xdf#8]
![image](https://cloud.githubusercontent.com/assets/303797/23286185/0f56c50c-fa04-11e6-9783-fb5103c269c4.png)

This seems a bit crazy. The sample data file is (unzipped) 693MB. I think we can probably get away with a less than 10 MB sample data file.

The version eeglab provides is 1.13 but the github repo last's release is 1.12. I don't know how to get a release with a different datafile to the eeglab repo nor do I know how to get the eeglab plugin download tool to point to the github release.

**Comment by arnodelorme** [comment]
Yes, this needs to be fixed. My version of XDF import is 113Kb (and it includes the binaries). How did you get that message. I just tried and it was 113kb?
